src/htmlminifier.js

```diff
@@ -481,18 +481,19 @@
   }
 
   function canRemoveElement(tag, attrs) {
-    if (tag === 'textarea') {
-      return false;
-    }
-
-    if (tag === 'script') {
-      for (var i = attrs.length - 1; i >= 0; i--) {
-        if (attrs[i].name === 'src') {
-          return false;
+    switch (tag) {
+      case 'audio':
+      case 'iframe':
+      case 'textarea':
+      case 'video':
+        return false;
+      case 'script':
```

Comment:
Shouldn't video, audio and iframe act the same way as script? Only be removed when they don't have "src" attr?